src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests.cs

```diff
@@ -44,17 +44,17 @@ static void Test2(int y, params int[] x)
             var nodes = tree.GetRoot().DescendantNodes().OfType<InvocationExpressionSyntax>().ToArray();
 
             compilation.VerifyOperationTree(nodes[0], expectedOperationTree:
-@"IInvocationExpression (static void Cls.Test1(params System.Int32[] x)) (OperationKind.InvocationExpression, Type: System.Void)
-  IArgument (Matching Parameter: x) (OperationKind.Argument)
-    IConversionExpression (ConversionKind.Cast, Implicit) (OperationKind.ConversionExpression, Type: System.Int32[], Constant: null)
-      ILiteralExpression (Text: null) (OperationKind.LiteralExpression, Type: null, Constant: null)");
+@"IInvocationExpression (static void Cls.Test1(params System.Int32[] x)) (OperationKind.InvocationExpression, Type: System.Void) (Syntax: 'Test1(null)')
+  Arguments(1): IArgument (Matching Parameter: x) (OperationKind.Argument) (Syntax: 'null')
+      IConversionExpression (ConversionKind.Cast, Implicit) (OperationKind.ConversionExpression, Type: System.Int32[], Constant: null) (Syntax: 'null')
+        ILiteralExpression (Text: null) (OperationKind.LiteralExpression, Type: null, Constant: null) (Syntax: 'null')");
 
             compilation.VerifyOperationTree(nodes[1], expectedOperationTree:
-@"IInvocationExpression (static void Cls.Test2(System.Int32 y, params System.Int32[] x)) (OperationKind.InvocationExpression, Type: System.Void, IsInvalid)
-  IArgument (Matching Parameter: y) (OperationKind.Argument)
-    IObjectCreationExpression (Constructor: System.Object..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Object)
-  IArgument (Matching Parameter: x) (OperationKind.Argument)
-    ILiteralExpression (Text: null) (OperationKind.LiteralExpression, Type: null, Constant: null)");
+@"IInvocationExpression (static void Cls.Test2(System.Int32 y, params System.Int32[] x)) (OperationKind.InvocationExpression, Type: System.Void, IsInvalid) (Syntax: 'Test2(new o ... ct(), null)')
+  Arguments(2): IArgument (Matching Parameter: y) (OperationKind.Argument) (Syntax: 'new object()')
```

Comment:
>Arguments(2): IArgument (Matching Parameter: y) (OperationKind.Argument) (Syntax: 'new object()') [](start = 2, length = 97)

It might be more readable to start the first argument on the new line.